inference-engine/src/gna_plugin/descriptions/gna_input_desc.cpp

```diff
@@ -29,3 +30,22 @@ float GNAPluginNS::InputDesc::getScaleFactor(const std::size_t index) {
     }
     return inputScaleFactors[index];
 }
+
+std::vector<GNAModelSerial::RuntimeEndPoint> GNAPluginNS::InputDesc::toEndPoints(const InferenceEngine::InputsDataMap& inputsDataMap) {
```

Comment:
disagree - encdpoint should be isolated feature solely related to serialisation - move it to serialisation as ctor of endpoint file and construct endpoint from inputsInfo - not from map - 
iteration can be don in plugin-serial